modules/api/php/models/candidatesrow.class.inc

```diff
@@ -63,7 +63,6 @@ class CandidatesRow implements \LORIS\Data\DataInstance,
         $obj = [
             'CandID'    => $this->_candid,
             'Project'   => $this->_projectname,
-            'ProjectID' => $this->_projectid,
```

Comment:
only removed it from json serialize. I kept the class variable because its used for permission checks